rewrite-core/src/main/java/org/openrewrite/scm/AzureDevOpsCloneUrl.java

```diff
@@ -18,23 +18,22 @@
 import lombok.Value;
 
 @Value
-public class AzureDevopsCloneUrl implements CloneUrl {
+public class AzureDevOpsCloneUrl implements CloneUrl {
     String cloneUrl;
     String origin;
     String path;
     String organization;
-    String project;
     String repositoryName;
 
-    public AzureDevopsCloneUrl(String cloneUrl, String origin, String path) {
+    public AzureDevOpsCloneUrl(String cloneUrl,
+                               String origin,
+                               String organization,
+                               String project,
+                               String repositoryName) {
         this.cloneUrl = cloneUrl;
         this.origin = origin;
-        this.path = path;
-        if (!path.contains("/")) {
```

Comment:
I think we should still do this check. Otherwise you get IndexOutOfBounds on a bad url